### PR TITLE
Change the method of obtaining tokens from implicit flow to pkce

### PR DESCRIPTION
**Pull request by zhichengliu12581**
Change the method of obtaining tokens from implicit flow to pkce by use Custom method replace msal-angular.js

### CODE REVIEW

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory/src/main/resources/static/App/Scripts/app.js [hunk 1]
```diff
@@ -15,20 +15,45 @@ angular.module('todoApp', ['ngRoute', 'MsalAngular'])
         }).otherwise({redirectTo: "/Home"});
 
         window.applicationConfig = {
-            clientID: 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx'
+            clientID: '762eb4c6-0597-4585-a3cb-198827bdef0c'
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory/src/main/resources/static/App/Scripts/app.js:18]: Oh... please delete this.
And delete related client for save purpose.

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory/src/main/resources/static/App/Scripts/app.js [hunk 2]
```diff
@@ -15,20 +15,45 @@ angular.module('todoApp', ['ngRoute', 'MsalAngular'])
...
             {
-                authority: 'https://login.microsoftonline.com/xxxorg.onmicrosoft.com',
-                clientID: applicationConfig.clientID,
-                cacheLocation: 'localStorage',
-                postLogoutRedirectUri: 'http://localhost:8080/logout',
-
-                tokenReceivedCallback: function (errorDesc, token, error, tokenType) {
+                auth: {
+                    clientId: "762eb4c6-0597-4585-a3cb-198827bdef0c",
+                    authority: "https://login.microsoftonline.com/7bf345ec-5c80-41a1-94ad-07ff910be5d8",
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory/src/main/resources/static/App/Scripts/app.js:25]: Same here.

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory/src/main/resources/static/index.html [hunk 3]
```diff
@@ -64,7 +64,7 @@ <h1>Todo List</h1>
 <script src="https://code.angularjs.org/1.2.25/angular-route.js"></script>
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
 <script src="App/Scripts/msal.js"></script>
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory/src/main/resources/static/index.html:66]: I remember that `msal.js` is not used in the application. Then can this line be deleted?

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory/src/main/resources/static/index.html [hunk 4]
```diff
@@ -64,7 +64,7 @@ <h1>Todo List</h1>
...
-<script src="App/Scripts/msal-angular.js"></script>
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory/src/main/resources/static/index.html:67]: Can we delete file `msal-angular.js` ?

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory/src/main/resources/static/index.html (hunk 3 above)
Comment (lzc-1997-abel) [reply sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory/src/main/resources/static/index.html:66]: we can delete `msal-angular.js` , but `msal.js` is used in my own function to obtaining tokens for "Authorization Code Flow with PKCE"

Comment (chenrujun) [reply sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory/src/main/resources/static/index.html:66]: I got it. Thank you for your explanation. 👍

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**